Added offence levels and chilling (#268)
w3f/polkadot-wiki: docs/polkadot/node/guides/how-to-set-up-secure-validate.md

# Secure Validator Setup Guide

> This article is based on the [polkadot-secure-validator](https://github.com/w3f/polkadot-secure-validator) repository to write and assumes you have already set up a Polkadot / Kusama validator. If not, please go [here](https://guide.kusama.network/en/latest/try/validate/) first.

Reviewer: ```suggestion
The following article is based on the [polkadot-secure-validator](https://github.com/w3f/polkadot-secure-validator) repository and assumes you have already set up a Polkadot or Kusama validator. If do not have any validator set-up experience, please follow the guide [here](https://guide.kusama.network/en/latest/try/validate/) for an introductory tutorial first.
```

> In this guide, we will enhance the security of the validator by setting up public nodes, as well as configure a VPN to make your validator communicate only with the public nodes.

Reviewer: ```suggestion
In this guide, we will enhance the security of the validator by setting a layered architecture with public nodes and a private validator node. The validator is configured on a VPN with the public nodes so that it's only visible to its own public nodes for communication.
```

2019-09-05 18:17:14 Applying authority set change scheduled at block #2207744
2019-09-05 18:17:14 Applying GRANDPA set change to new set [(3a92077b16fbb87972be7ebaf1b7e70f5b4fac9636c136936a28d0fb494d1ed4 (5DPW1n4q...), 1), (4bd3620064cda1f4cf405bf9ab565c9bad69446034c48884ffc5363a5286b145 (5Dn8F1SU...), 1), (ca8feb6f870330cdaea24e49c2f850b66729340cab164aea86c0a782ddecf57a (5GeJHN5E...), 1), (dcb83e46917c3c0ca35b9a18a32ba6d3912b6d50ab2bd382341d2e4fd2e6946f (5H4787dX...), 1)]
2019-09-05 18:17:15 Imported justification for block #2207744 that triggers command Changing authorities, signaling voter.
2019-09-05 18:17:17 Syncing 53.2 bps, target=#2953887 (1 peers), best: #2207882 (0x217a…f96a), finalized #2207744 (0xcdf8…6cee), ⬇ 50.3kiB/s ⬆ 15.8kiB/s
2019-09-05 18:17:18 Discovered new external address for our node: /ip4/10.0.1.203/tcp/30333/p2p/QmaiebwiaaaFRpAd7DTR8bv72BJs9WAMLWbjP61JeKpFf5
```

Based on the above output, there is only 1 peer connected to the validator.  This is the public node we defined by using the `--reserved-nodes` flag to specify the public node internal address and the node identity.

## Summary

So far you have successfully:

> - Configured the firewall

Reviewer: ```suggestion
- Configured the firewall.
```

> - Set up and configured the Wireguard

Reviewer: ```suggestion
- Set up and configured the Wireguard.
```

> - Ran a validator with public node

Reviewer: ```suggestion
- Ran a validator with public nodes.
```

> If you have experience with devops tools such as `Terraform` and `Ansible`, please check out the [secure validator setup](https://github.com/w3f/polkadot-secure-validator) repository. This tool does everything we talked about above, but you still need to define some credentials(e.g. IP of your server) in the inventory file and then you can manage the relevant servers in an automated way. 

Reviewer: ```suggestion
If you have experience with devops tools such as `Terraform` and `Ansible`, please check out the [secure validator setup](https://github.com/w3f/polkadot-secure-validator) repository. This tool does everything we talked about above, but you still need to define some credentials (e.g. IP of your server) in the inventory file and then you can manage the relevant servers in an automated way. 
```

> Note: Sentry Node in the networking layer has not been fully implemented yet, so please stay tuned. We will keep updating the guide with any updates on this [issue](https://github.com/paritytech/substrate/issues/2999).

> ## Todos

Reviewer: Let's remove these Todos before we merge.
